vscode-extension/src/aiConfigEditor.ts

```diff
@@ -338,7 +373,24 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
     await waitUntilServerReady(editorServer.url);
 
     // Now set up the server with the latest document content
-    await initializeServerState(editorServer.url, document);
+    try {
+      await updateServerState(editorServer.url, document);
+    } catch (e) {
+      vscode.window
+        .showErrorMessage(
+          "Failed to start aiconfig server. You can view the aiconfig but cannot modify it.",
+          ...["Details", "Retry"]

```

Comment:
What does the "Details" here add? Looks the same in test plans

Reply:
It adds the 'Details' button in the notification, clicking it opens the Output tab with the exception message